examples/node/getinfo.js

```diff
@@ -8,7 +8,7 @@
 //
 
 // Run `gulp dist-install` to generate 'pdfjs-dist' npm package files.
-var pdfjsLib = require("pdfjs-dist");
+var pdfjsLib = require("pdfjs-dist/es5/build/pdf.js");
```

Comment:
What is the reason for using ES5-compatible builds for the examples? Wouldn't it be better to switch them to the non-ES5-compatible builds so we promote that instead?

Reply:
Unfortunately Node.js still requires *some* polyfills (e.g. `ReadableStream`), which is why this is done.
However, the other (browser) examples are using the "correct" file, although I probably ought to update all of the existing ones to add a *commented out* line with the es5 version for documentation purposes.

Reply:
Unfortunately Node.js still requires *some* polyfills, e.g. `ReadableStream`, which is why this is done.